From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Xiaoyin Liu <xiaoyinl@users.noreply.github.com>
Date: Sun, 6 Jun 2021 17:20:13 +0800
Subject: [PATCH] 3109: Remove symbol servers that don't have PDB symbols

These symbol servers (Intel, Nvidia, and AMD) only provide dll files, not PDB files.
---
 .../Public_Release/data/PDB_SYMBOL_SERVER_URLS.pdburl          | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/Ghidra/Configurations/Public_Release/data/PDB_SYMBOL_SERVER_URLS.pdburl b/Ghidra/Configurations/Public_Release/data/PDB_SYMBOL_SERVER_URLS.pdburl
index 06da4577c..33c2aa606 100644
--- a/Ghidra/Configurations/Public_Release/data/PDB_SYMBOL_SERVER_URLS.pdburl
+++ b/Ghidra/Configurations/Public_Release/data/PDB_SYMBOL_SERVER_URLS.pdburl
@@ -1,6 +1,3 @@
 Internet|https://msdl.microsoft.com/download/symbols/|WARNING: Check your organization's security policy before downloading files from the internet.
 Internet|https://chromium-browser-symsrv.commondatastorage.googleapis.com|WARNING: Check your organization's security policy before downloading files from the internet.
 Internet|https://symbols.mozilla.org/|WARNING: Check your organization's security policy before downloading files from the internet.
-Internet|https://software.intel.com/sites/downloads/symbols/|WARNING: Check your organization's security policy before downloading files from the internet.
-Internet|https://driver-symbols.nvidia.com/|WARNING: Check your organization's security policy before downloading files from the internet.
-Internet|https://download.amd.com/dir/bin|WARNING: Check your organization's security policy before downloading files from the internet.
-- 
2.43.0

